frontends/web/src/components/appupgraderequired.tsx

```diff
@@ -15,27 +15,26 @@
  * limitations under the License.
  */
 
-import { translate, TranslateProps } from '../decorators/translate';
-import { AppDownloadLink } from './appdownloadlink/appdownloadlink';
+import { useTranslation } from 'react-i18next';
+import { View, ViewButtons, ViewHeader } from './view/view';
+import { AppDownloadButton } from './appdownloadlink/appdownloadlink';
+import { Header, Main } from './layout';
 
-function AppUpgradeRequired({ t }: TranslateProps) {
+export const AppUpgradeRequired = () => {
+  const { t } = useTranslation();
   return (
-    <div className="contentWithGuide">
-      <div className="container">
-        <div className="innerContainer">
-          <div className="content narrow isVerticallyCentered">
-            <div className="box large">
-              <p className="m-top-none">{t('device.appUpradeRequired')}</p>
-              <div className="buttons m-top-half">
-                <AppDownloadLink className="text-medium text-blue" />
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
+    <Main>
+      <Header />
+      <View
+        fullscreen
+        verticallyCentered
+        width="840px"
```

Comment:
May be better that one day we have these (and media query values) to be using variables instead of hardcoded.

Reply:
yes makes sense, currently it's ok I think, there is a `min-width: 100%` so this prop is more like desired width but can go smaller.

Maybe like this: `style={{ "--content-width-large": "840px", "--content-width-small": "640px" }}`?